src/main/java/club/sk1er/patcher/mixins/bugfixes/GuiScreenResourcePacksMixin_FixOpenPackFolder.java

```diff
@@ -0,0 +1,29 @@
+package club.sk1er.patcher.mixins.bugfixes;
+
+import gg.essential.universal.UDesktop;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.GuiScreenResourcePacks;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(GuiScreenResourcePacks.class)
+public class GuiScreenResourcePacksMixin_FixOpenPackFolder {
+
+    @Inject(
+        method = "actionPerformed",
+        at =
+        @At(
+            value = "INVOKE",
+            target = "Lnet/minecraft/client/resources/ResourcePackRepository;getDirResourcepacks()Ljava/io/File;",
+            shift = At.Shift.BY,
+            by = 2),
```

Comment:
Why the shift?

Reply:
because the MethodInsnNode targeted is followed by an ASTORE, and if I inject directly after, it splits the local variable allocation in two and in the decompiled class it looks ugly. It makes the decompiler invent a fake local variable
![image](https://github.com/Sk1erLLC/Patcher/assets/57050655/e54d53b8-830b-4260-bc58-1cf41a13fa1c)

Reply:
with the shift by 2 : 
![image](https://github.com/Sk1erLLC/Patcher/assets/57050655/479a8178-861b-4380-aad3-bf149872d6da)

Reply:
This doesn't justify using a shift, decompiled code will always be messy.

Reply:
I can remove it if you'd like